R/textstat_simil.R

```diff
@@ -145,7 +145,7 @@ textstat_dist.dfm <- function(x, selection = NULL,
 #' [Experimental] Compute document/feature proximity
 #'
 #' This is an underlying function for \code{textstat_dist} and
-#' \code{textstat_simil} but returns \code{CsparaseMatrix}. 
+#' \code{textstat_simil} but returns \code{sparseMatrix}. 
```

Comment:
Technically, `sparseMatrix` is a convenience function for creating one of the sparse Matrix classes, but this is probably fine too.